app/state_machines/configurable_state_machine/machine.rb

```diff
@@ -188,12 +191,17 @@ def find_destination_state(event_config:)
       event_config.to_h.key?(:transition_to) ? event_config.transition_to : @kase.current_state
     end
 
-    def write_transition(event:, to_state:, params:)
+    def find_destination_workflow(event_config:)
+      event_config.to_h.key?(:switch_workflow) ? event_config.switch_workflow : @kase.workflow
+    end
+
+    def write_transition(event:, to_state:, to_workflow:, params:)
       attrs = {
         event: event,
-               to_state: to_state,
-               sort_key: CaseTransition.next_sort_key(@kase),
-               most_recent: true,
+        to_state: to_state,
+        to_workflow: to_workflow,
```

Comment:
If you're making changes anyway, run `annotate` so this column is added to the annotations ... I was looking for it when I saw this. :)

Reply:
done